### DISCUSSION TITLE
Landafræðislagurinn - 2024/4/4

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#236] [vefsida-ummæli]
# Landafræðislagurinn - 2024/4/4

Heimasíða Biskupsreglunnar

https://www.biskupsreglan.com/frettir/Landafr%C3%A6%C3%B0islagurinn

**Comment by TJ478** [comment]
Slagur!!! Slagur!!! Slagur !!!
